src/snowflake_utils/README.md

```diff
@@ -0,0 +1,92 @@
+# Snowflake Utils
+
+## How to Use
+
+The functions in these scripts are meant to be imported into relevant DAGs to make it easier to connect to Snowflake when you want to have separate custom modules to write code that involves Snowflake outside of the DAGs themselves
+
+
+## Example Usage
+
+### Connecting to snowflake locally in your code
+
+Setup your environment variables either by executing the following commands directly in your terminal:
+
+```
+export SNOWFLAKE_USER=
```

Comment:
Future potential improvement: consider using .dotenv package

Reply:
I originally wanted to include it but also wanted to avoid having one more pkg to maintain in the requirements-airflow.txt. Agree that .dotenv is easier to use.